WordPress/Classes/ViewRelated/Me/App Settings/Boolean User Defaults/BooleanUserDefaultsDebugViewModel.swift

```diff
@@ -0,0 +1,94 @@
+import SwiftUI
+
+final class BooleanUserDefaultsDebugViewModel: ObservableObject {
+    @Published private var allUserDefaultsSections = BooleanUserDefaultsSections()
+    @Published var searchQuery: String = ""
+
+    private var persistentRepository: UserPersistentRepository
+
+    var userDefaultsSections: BooleanUserDefaultsSections {
+        return if searchQuery.isEmpty {
+            allUserDefaultsSections
+        } else {
+            filterUserDefaults(by: searchQuery)
+        }
+    }
+
+    let coreDataStack: CoreDataStack
+
+    private func filterUserDefaults(by query: String) -> BooleanUserDefaultsSections {
```

Comment:
[Nit]:
```suggestion
    private func filterUserDefaults(by query: String) -> BooleanUserDefaultsSections {
        guard !searchQuery.isEmpty else {
            return allUserDefaultsSections
        }
```
This way `userDefaultsSections` can be simplified to just `filterUserDefaults(by: searchQuery)`